src/rmq/CMakeLists.txt

```diff
@@ -38,6 +42,17 @@ install(
         COMPONENT librmq-dev
 )
 
+install(EXPORT rmqcppTargets 
+        FILE rmqcppTargets.cmake 
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/share/rmqcpp 
+        NAMESPACE rmqcpp::)
+
+include(CMakePackageConfigHelpers)
+configure_package_config_file(${CMAKE_CURRENT_SOURCE_DIR}/Config.cmake.in 
```

Comment:
This uses `Config.cmake.in` as input and produces the `rmqcppConfig.cmake`  which `find_package` looks for.